vllm/config.py

```diff
@@ -1356,20 +1371,24 @@ def verify_with_parallel_config(
             logger.warning("Possibly too large swap space. %s", msg)
 
 
+PoolType = Literal["ray"]
+
+
+@config
 @dataclass
 class TokenizerPoolConfig:
-    """Configuration for the tokenizer pool.
-
-    Args:
-        pool_size: Number of tokenizer workers in the pool.
-        pool_type: Type of the pool.
-        extra_config: Additional config for the pool.
-            The way the config will be used depends on the
-            pool type.
-    """
-    pool_size: int
-    pool_type: Union[str, type["BaseTokenizerGroup"]]
-    extra_config: dict
+    """Configuration for the tokenizer pool."""
+
+    pool_size: int = 0
+    """Number of tokenizer workers in the pool to use for asynchronous
+    tokenization. If 0, will use synchronous tokenization."""
+    pool_type: Union[PoolType, type["BaseTokenizerGroup"]] = "ray"
+    """Type of tokenizer pool to use for asynchronous tokenization. Ignored if
+    tokenizer_pool_size is 0."""
+    extra_config: dict = field(default_factory=dict)
+    """Additional config for the pool. The way the config will be used depends
+    on the pool type. This should be a JSON string that will be parsed into a
+    dictionary. Ignored if tokenizer_pool_size is 0."""
```

Comment:
```suggestion
    pool_size: int = 0
    """Number of tokenizer workers in the pool to use for asynchronous
    tokenization. If 0, will use synchronous tokenization."""

    pool_type: Union[PoolType, type["BaseTokenizerGroup"]] = "ray"
    """Type of tokenizer pool to use for asynchronous tokenization. Ignored if
    tokenizer_pool_size is 0."""

    extra_config: dict = field(default_factory=dict)
    """Additional config for the pool. The way the config will be used depends
    on the pool type. This should be a JSON string that will be parsed into a
    dictionary. Ignored if tokenizer_pool_size is 0."""
```
Add a new line between each field for readability